cloud/migrate-to-cloud/schema-then-data.md

```diff
@@ -140,7 +140,7 @@ After pre-data migration, your hypertables from your source database become
 regular PostgreSQL tables in Cloud. Recreate your hypertables in Cloud to
 restore them.
 
-<procedure>
+<Procedure>
```

Comment:
I wonder if the angle brackets in L134 are breaking something here? (maybe it's only breaking GHs highlighting, though)

Reply:
Just checked in preview and it looks like a GH-highlighting thing